vividus-plugin-web-app-to-rest-api/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -68,6 +67,25 @@
         <property name="filterRedirects" value="${transformer.from-sitemap.filter-redirects}" />
         <property name="mainPageUrl" value="${transformer.from-sitemap.main-page-url}" />
         <property name="mainPageUrlProperty" value="transformer.from-sitemap.main-page-url" />
+        <property name="httpRedirectsProvider">
+            <bean class="org.vividus.http.HttpRedirectsProvider">
+                <property name="httpClient">
```

Comment:
but we already have http client for `CrawlerCommonsSiteMapParser`, can't we reuse it?